app/tasks/schedule.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+global $cli;
+global $register;
+
+use Cron\CronExpression;
+use Utopia\App;
+use Utopia\CLI\Console;
+use Utopia\Database\DateTime;
+use Utopia\Database\Query;
+use Swoole\Timer;
+
+const FUNCTION_UPDATE_TIMER = 60; //seconds
+const FUNCTION_ENQUEUE_TIMER = 60; //seconds
+const ENQUEUE_TIME_FRAME = 60 * 5; // 5 min
+sleep(4); // Todo prevent PDOException
```

Comment:
Can we remove this now ?

Reply:
sure

Reply:
When you remove it you get every 2-3 times a PDO missing table errors.
I guess it  is a race condition somewhere

PDOException: Table 'appwrite.console__metadata' doesn't exist in @swoole-src/library/core/Database/PDOStatementProxy.php:64
Stack trace: